test/integration/test/pagekite_ping_test.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const spawn = require('child_process').spawn;
+const chakram = require('chakram'), expect = chakram.expect;
+const Config = require('config-js');
+
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var header = new Config('./test/integration/lib/config/header.js');
+var philipshue_server = require('../lib/philipsHue_server.js');
+var nupnp_server = require('../lib/nupnp_PhilipsHue.js');
+
+describe('Initiate the remote connection',function(){
+  var local_ip,tunnel_url;
+  var foxbox_process;
+  var credential = config.get('credential'); 
+  
+  before(function(done){
+    var FOXBOX_STARTUP_WAIT_TIME_IN_MS = 3000;
+    this.timeout(500000);
+
+    // start nupnp server
+    nupnp_server.start(config.get('nupnp_server.id'),
+      config.get('philips_hue.url') +':'+config.get('philips_hue.port'),
+      config.get('nupnp_server.port'));
+    
+    // configure philips hub simulator
+    philipshue_server.setup(config.get('philips_hue.port'));
+
+    // start foxbox
+    foxbox_process = spawn('./target/debug/foxbox', 
+      ['-c',  config.get('nupnp_server.param')+';'+ 
+      config.get('nupnp_server.url')+':'+ 
+      config.get('nupnp_server.port')+'/',
+      '-r',config.get('pagekite.r'),
+      '-t',config.get('pagekite.t'),
+      '-s',config.get('pagekite.s'),
+      '--remote-name',config.get('pagekite.externalURL'),
+      '--disable-tls'] );
+
+    // give time until foxbox is operational
+    setTimeout(done, FOXBOX_STARTUP_WAIT_TIME_IN_MS);  
+  });
+  
+  after(function(){
+    foxbox_process.kill('SIGINT');
+    require('../lib/foxboxHelper.js').removeDB();
```

Comment:
Maybe this could also be incoporated in [FoxboxManager](https://github.com/fxbox/foxbox/pull/235/files#r56685935)